corgidrp/data.py

```diff
@@ -847,11 +854,11 @@ def __init__(self, data_or_filepath, err = None, ptc = None, pri_hdr=None, ext_h
         if 'RN_ERR' not in self.ext_hdr:
             self.ext_hdr['RN_ERR'] = ''
         # File format checks
-        if self.data.shape != (1,1):
```

Comment:
We may need to check if this will affect anyone other than the DRP (e.g., does GITL use this?), as we don't want to break compatibility.